owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/status/GetRemoteCapabilitiesOperation.kt

```diff
@@ -45,211 +50,39 @@ import timber.log.Timber
  * @author masensio
  * @author David González Verdugo
  */
-/**
- * Constructor
- */
 class GetRemoteCapabilitiesOperation : RemoteOperation<RemoteCapability>() {
 
     override fun run(client: OwnCloudClient): RemoteOperationResult<RemoteCapability> {
         var result: RemoteOperationResult<RemoteCapability>
 
         try {
-            val requestUri = client.baseUri
-            val uriBuilder = requestUri.buildUpon()
-            uriBuilder.appendEncodedPath(OCS_ROUTE)    // avoid starting "/" in this method
-            uriBuilder.appendQueryParameter(PARAM_FORMAT, VALUE_FORMAT)
-
-            val getMethod = GetMethod(URL(uriBuilder.build().toString()))
-
-            getMethod.addRequestHeader(OCS_API_HEADER, OCS_API_HEADER_VALUE)
-
+            val uriBuilder = client.baseUri.buildUpon().run {
```

Comment:
Why do you use `apply` below and `run` here? In both cases you want to return the object itself, so I would use `apply` for both. `run` fits better when you want to do something else with the receiver and return a result.

Reply:
My bad, updated 👍